core/src/test/scala/unit/kafka/server/ReplicaAlterLogDirsThreadTest.scala

```diff
@@ -458,7 +458,7 @@ class ReplicaAlterLogDirsThreadTest {
       ArgumentCaptor.forClass(classOf[org.apache.kafka.server.common.TopicIdPartition])
     val logIdCaptureT1p0: ArgumentCaptor[Uuid] = ArgumentCaptor.forClass(classOf[Uuid])
 
-    verify(directoryEventHandler).handleAssignment(topicIdPartitionCaptureT1p0.capture(), logIdCaptureT1p0.capture(), any())
+    verify(directoryEventHandler).handleAssignment(topicIdPartitionCaptureT1p0.capture(), logIdCaptureT1p0.capture(), "test", any())
```

Comment:
Looks like this change failed tests:

https://ci-builds.apache.org/job/Kafka/job/kafka-pr/job/PR-15522/2/testReport/junit/kafka.server/ReplicaAlterLogDirsThreadTest/Build___JDK_11_and_Scala_2_13___shouldRevertAnyScheduledAssignmentRequestIfAssignmentIsCancelled__/

Maybe `anyString()`?

Reply:
Thanks for pointing this out. Fixed the test and also resolved the conflicts.